toolkit/tools/imagecustomizer/docs/iso.md

```diff
@@ -98,4 +104,20 @@ os:
     cloud-init-data/user-data: /cloud-init-data/user-data
     cloud-init-data/network-config: /cloud-init-data/network-config
     cloud-init-data/meta-data: /cloud-init-data/meta-data
-```
\ No newline at end of file
+```
+
+## Input Image Layout Assumptions
+
+### Full Disk Image
+
+The input full disk image must satisfy the following requirements in order for
+MIC to be able to generate an iso image out of it:
+- File layout (after all partitions have been mounted):
+  - `/boot/grub2/grub.cfg` must exist and is the 'main' grub configuration (not
+    a redirection grub configuration file for example).
+  - `bootx64.efi` and `grubx64.efi` (or `grubx64-noprefix.efi`) must be under
+    the `/boot` folder (exact location is irrelevant).
```

Comment:
What does "exact location is irrelevant" mean? Are you saying that the files just have to be under "/boot" or any of its subdirectories?

Reply:
fixed.